### ISSUE TITLE
There is no 'after' field in addColumn third option (typescript)

### ISSUE THREAD

**Issue by pozniakas** [sequelize/sequelize#12754]
## Issue Description
I am just trying to make my migrations with typescript, but typescript shows the error, which should not appear:
![image](https://user-images.githubusercontent.com/50362698/95734626-64247e00-0c8c-11eb-88fd-6d1c3a2d4f3c.png)

### What do you expect to happen?

I expect to not get TypeScript error here, because everything works fine

### What is actually happening?

I got an error

### Environment

- Sequelize version: XXX  5.8.12
- Node.js version: XXX  12-alpine
- Operating System: Linux 
- If TypeScript related: TypeScript version: 3.9.7

**Comment by Zikoel** [comment]
In another issue I saw that a possible workaround before the fix can be put the option and skip the ts error with `// @ts-ignore` is not very good but it's compile and the option is working!

**Comment by dpozniakas** [comment]
Yeah, I agree @Zikoel. This is what I'm doing now, but it sucks to do that every time, when I use `after` key :D

**Comment by pozniakas** [comment]
Still waiting for fix

**Comment by lcundiff** [comment]
Still giving typescript error.

**Comment by kottofey** [comment]
Yaaay! Still no typescript fix ))

As a workaround, youmay add a `.d.ts` file with this content:
```ts
declare module 'sequelize' {
  interface ModelAttributeColumnOptions {
    /**
     * A column after which a new column must be inserted
     */
    after: string;
  }
}

export {};
```

Do not forget to include it in project's `tsconfig.json`. So no more such dirty hacks as `// @ts-ignore` needed.


PS. I hope in Sequelize 7 this would be fixed...
